src/pages/EventDashboard/EventDashboard.jsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react'
+import EventsList from '../../components/EventList/EventList'
+import CreateEventForm from '../../components/CreateEventForm/CreateEventForm'
+import EditEventForm from '../../components/EditEventsForm/EditEventsForm'
+import EventCrud from '../../components/EventCrud/EventCrud'
+import useAuth from '../../hooks/use-auth'
+import useSelf from '../../hooks/use-self';
+import './EventDashboard.css'
+
+
+
+function EventDashboard() {
+
+    const { auth, setAuth } = useAuth();
+    const { self, isLoading, error } = useSelf();
+    const [activeEvent, setActiveEvent] = useState("")
+    const [createEventOpen, setCreateEventOpen] = useState("false")
+    const [editEventOpen, setEditEventOpen] = useState("false")
+
+    // Creating events
+    const onCreateEventClick = (event) => {
+        setCreateEventOpen(event)
+    }
+
+    // Editing events
+    const onEditEventClick = (event) => {
+        setEditEventOpen(event)
+    }
+
+    // Current event view
+    const onChangeActiveEvent = (eventID) => {
+        setActiveEvent(eventID)
+    }
+
+    return (
+        <>
+            {auth.token && self?.is_staff ?
+                <div className='dashboard'>
+
+                    <EventsList onEditEventClick={onEditEventClick} activeEvent={activeEvent} onChangeActiveEvent={onChangeActiveEvent} onCreateEventClick={onCreateEventClick} createEventOpen={createEventOpen} />
+
+                    {createEventOpen === "true" ?
+                        <CreateEventForm onCreateEventClick={onCreateEventClick} createEventOpen={createEventOpen} onChangeActiveEvent={onChangeActiveEvent} activeEvent={activeEvent} />
+                        :
+                        <>
+                            {editEventOpen === "true" ?
+                                <EditEventForm onEditEventClick={onEditEventClick} editEventOpen={editEventOpen} onCreateEventClick={onCreateEventClick} createEventOpen={createEventOpen} onChangeActiveEvent={onChangeActiveEvent} activeEvent={activeEvent} />
+                                :
+                                <>
+                                    {activeEvent != '' ?
+                                        <EventCrud activeEvent={activeEvent} onCreateEventClick={onCreateEventClick} createEventOpen={createEventOpen}
+                                            editEventOpen={editEventOpen} onEditEventClick={onEditEventClick} />
+                                        :
+                                        <p>Find/click on an event, or create a new event</p>
+
+                                    }
+                                </>
+                            }
+                        </>
+                    }
+                </div>
+                :
+                <div className='forbidden'>
```

Comment:
This looks ridiculously amazing, however, we do already have a 404 page.
I recommend either:
a) using the existing 404 page or
b) updating the existing 404 page to have this content instead, then call on this 404 page component here

Reply:
Will swap to call the existing 404 - and we can check in with team on how we want it to look :D